server/src/http/controllers/application.controller.ts

```diff
@@ -15,6 +16,56 @@ const rateLimitConfig = {
 } as const
 
 export default function (server: Server) {
+  server.post(
+    "/v1/application",
+    {
+      schema: zRoutes.post["/v1/application"],
+      config: {
+        rateLimit: {
+          max: 5,
+          timeWindow: "5s",
+        },
+      },
+      bodyLimit: 5 * 1024 ** 2, // 5MB
+    },
+    async (req, res) => {
+      const { company_type } = req.body
+
+      const convertedCompanyType = () => {
+        switch (company_type) {
+          case "matcha":
+            return LBA_ITEM_TYPE.OFFRES_EMPLOI_LBA
+          case "lba":
+          case "lbb":
+            return LBA_ITEM_TYPE.RECRUTEURS_LBA
+
+          case "offres":
+            return LBA_ITEM_TYPE.OFFRES_EMPLOI_PARTENAIRES
+
+          default:
+            return
+        }
+      }
+
+      const result = await sendApplication({
+        newApplication: { ...req.body, company_type: convertedCompanyType() as string },
```

Comment:
```suggestion
        newApplication: { ...req.body, company_type: convertedCompanyType() },
```
le `as string`  ne devrait pas être nécessaire si la fonction renvoi un résultat correct